### DISCUSSION TITLE
Force OCR on specific documents after importing

### DISCUSSION THREAD

**Discussion by 5hay** [jonaswinkler/paperless-ng#808] [Feature Requests]
Hi,

I would love to have a button in the UI to force / redo an OCR on specific documents. Either by selecting multiple documents on the `/documents` page and/or on the `documents/<id>` page.

_Why?_

Some older documents (that I've scanned with my smartphone) have bad OCR and paperless-ng is set to skip OCR on already OCR'd documents. I don't see any way to force it on those specific documents.

**Comment by jonaswinkler** [comment]
Good idea. I'm actually in the same position but was simply too lazy to fix it. When I used paperless (not -ng), I used the OCR software of my scanner instead of tesseract, since that would add selectable text to the files. OCR quality is quite bad though.

I'll see what I can do.

**Reply by 5hay** [reply]
Thanks for the quick reply. So you're saying the OCR of paperless-ng is pretty bad?

If we can manually specify the documents that should be re-OCR'd it should be fine I think. One of my documents literally has like one recognized letter in every sentence, so it should be easy to have better results than this 😀

**Reply by jonaswinkler** [reply]
Oh, no. OCR quality of my scanner software is pretty bad. Tesseract (this is what paperless uses) does a fantastic job.

**Reply by 5hay** [reply]
Gotcha. I was confused for a moment :D

Thank you.
